package.json

```diff
@@ -68,6 +68,7 @@
   },
   "dependencies": {
     "lodash": "^4.17.4",
-    "mz": "^2.7.0"
+    "mz": "^2.7.0",
```

Comment:
I think we're missing a `devDependency` on `@types/uniqid`

Reply:
Ye we do, I'm pretty sure I added it but it somehow did not end up in the package.json aparantly :P